src/components/Post/Tabs/PostDescription.tsx

```
@@ -131,7 +131,7 @@ const PostDescription: FC<IPostDescriptionProps> = (props) => {
 			key: 4,
 			label: (
 				<div>
-					{allowed_roles && allowed_roles.includes('moderator') && isOffchainPost && ['polkadot', 'kusama', 'picasso', 'composable'].includes(network) && (
+					{((allowed_roles && allowed_roles.includes('moderator')) || canEdit) && isOffchainPost && ['polkadot', 'kusama', 'picasso', 'composable'].includes(network) && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)